📖 Docs: Installing clusterctl on windows section added (#6177)
kubernetes-sigs/cluster-api: docs/book/src/user/quick-start.md

clusterctl version
```

{{#/tab }}
{{#tab windows}}

> #### Install clusterctl binary with curl on windows

Reviewer: ```suggestion
#### Install clusterctl binary with curl on Windows
```

nit

Author: > nit

@killianmuldoon was earlier committing with "Windows" only but saw everything else in the doc is small for eg : linux, macOS, homebrew.

Should I make changes and commit ?

Go to the working directory where you want clusterctl downloaded.

> Download the latest release; on windows, type:

Reviewer 2: I wonder if we should explicitly state to run this in PowerShell. I don't believe this will work if someone uses cmd.exe.

Reviewer 2: That looks great, thanks! I think that would help avoid possible confusion later on if someone is not used to using PowerShell as their default console on Windows.

I agree with Killian, let's use proper casing for these: "Windows using PowerShell".

Author: on it

Author: > That looks great, thanks! I think that would help avoid possible confusion later on if someone is not used to using PowerShell as their default console on Windows.
> 
> I agree with Killian, let's use proper casing for these: "Windows using PowerShell".

Done :v:

> Download the latest release; on windows, type:

Reviewer: ```suggestion
Download the latest release; on Windows, type:
``

```

> curl {{#releaselink gomodule:"https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.1.1/clusterctl-windows-amd64.exe"}} -o clusterctl.exe

Reviewer 3: ```suggestion
curl {{#releaselink gomodule:"sigs.k8s.io/cluster-api" asset:"clusterctl-windows-amd64.exe" version:"1.1.x"}} -o clusterctl.exe
```

This will fix the pipeline errors you are seeing on the pipeline.

Author: Surely will commit this change, once everything is clear like what link to be added and all for setting env variables.
That will help to make all changes in one commit ig which will be better. :v:

```

> Add the path of that directory to path environment variable.

Reviewer 3: ```suggestion
Append or prepend the path of that directory to the `PATH` environment variable.
```

> Add the path of that directory to path environment variable.

Reviewer: Could you put an example of how to set the path here here? Or maybe just add a directory that's normally on the Path in Windows. That's what we're doing in the Linux / MacOS guides

Author: Yeah I was also thinking of adding command which is being used to add directory to environment path.
But didn't find anything. Everywhere I just saw the UI steps to add directory path to environment variables.

Reviewer 3: I am not a 100% sure - I will let you test this on a windows machine but I think you can add to the path in the current session using `set PATH=%PATH%;C:\your\path\here\` but to add it permanently you might have to use the GUI. 
If there is any standard documentation that shows how to add to the PATH using GUI then it would be nice to have that linked here.
